api/src/main/java/io/strimzi/api/kafka/model/KafkaRebalanceSpec.java

```diff
@@ -118,6 +119,15 @@ public void setReplicationThrottle(long bandwidth) {
         this.replicationThrottle = bandwidth;
     }
 
+    @Description("A list of strategies used to determine the execution order for the generated optimization proposal. By default BaseReplicaMovementStrategy is used, which is totally random.")
```

Comment:
Not a native speaker ... but _totally random_ sounds very informal for documentation. I also doubt it is really random - i.e. randomizing the list of movements ... I guess it might be just as it is generated by some algo or something like that I guess. Wouldn't something like `By default BaseReplicaMovementStrategy is used, which moves the replicas without any specific order.` be a bit better?

Reply:
Agreed with Jakub here, his wording looks good to me, or if it is truly random (or atleast different with the same input) - then it might be worth saying it is _non-deterministic_

Reply:
Sorry, this is copy of CC's wording, which I removed from other docstrings but didn't change here. I will change it.